lib/widget/evaluation_form/post-op-home/abnormal_symptom_form.dart

```diff
@@ -159,6 +154,23 @@ class _AbnormalSymptomFormState extends State<AbnormalSymptomForm> {
     );
   }
 
+  void alert(BuildContext context) {
+    showDialog(
+        context: context,
+        builder: (_) {
+          Future.delayed(Duration(seconds: 2), () {
+            Navigator.of(context).pop(true);
+          });
+          return AlertDialog(
+            shape:
+                RoundedRectangleBorder(borderRadius: BorderRadius.circular(15)),
+            title: Text("แสดงเครื่องหมาย √ \nในข้อที่ท่านมีอาการ",
```

Comment:
this word should be `ทำเครื่องหมาย` รึเปล่า

Reply:
หรืออย่างนี้ก็ได้แล้วแต่ ไม่ซี555555

Reply:
เออว่ะ ทำเครื่องหมายเข้าใจชัดกว่า เดี๋ยวลองเปลี่ยน